aten/src/TH/THRandom.cpp

```diff
@@ -10,14 +13,20 @@
 #define n _MERSENNE_STATE_N
 #define m _MERSENNE_STATE_M
 
+struct mutex_s {
```

Comment:
Could you please explain why adding a struct with only one member

Reply:
because `THRandom.h` isn't C++, I think. mutex_s is used there.